frontend/src/components/AccessGame/GameState.vue

```diff
@@ -13,47 +13,49 @@
       </div>
     </template>
     <template v-else>
-      <div v-if="game.status === 'created'" class="card">
+      <div v-if="game && game.status === 'created'" class="card">
         <div class="card-body">
           <h6>Invite other players to join</h6>
-          <div class="badge badge-pill badge-secondary" :class="{ 'badge-success': player.isHost }">{{ game.id }}</div>
-          <p v-if="players.length > 1" class="mt-3 mb-0">
+          <div class="badge badge-pill badge-secondary" :class="{ 'badge-success': !!(player || {}).isHost }">
+            {{ (game || {}).id }}
```

Comment:
I'd rather have the computed property return an empty object if game is not available to keep the template cleaner.

Reply:
We could, I propose to fix them in #99.